source/guides/clear/clear-linux-dissector.rst

```diff
@@ -0,0 +1,302 @@
+.. _clear-linux-dissector:
+
+Clear Linux\* Dissector
+#######################
+
+The Clear Linux\* Dissector tool compares contents of Linux-based images. It
+supports the comparison of |CL| and Yocto Project\* images.
+
+.. contents:: :local:
+   :depth: 1
+
+Description
+***********
+
+Clear Linux Dissector provides tools for comparing |CL| images and Yocto Project
+images.
+
+* **View distribution data.**
+
+  View the distribution data of a specific |CL| and Yocto Project distribution
+  side-by-side. Helpful to determine if a |CL| version contains the packages and
+  patches needed to move from a specific Yocto Project distribution to |CL|.
+
+* **Compare images.**
+
+  Compare |CL| and Yocto Project images to identify which packages you will need
+  to move from a Yocto Project image to a |CL| image.
+
+* **Compare releases.**
+
+  Compare different |CL| releases to see changes between |CL| versions.
+
+Prerequisites
+*************
+
+#. Must be on a Linux machine.
```

Comment:
@arbrindle - Need details regarding Prerequirements for setting up the tool

Reply:
@krisdale Correct, must be on any type of Linux machine, and must have docker, docker-compose, and python3

Reply:
There is one important pre-requisite for the YP image comparison part - if you want to collect the data from a YP image using the script downloaded from the app (under "Image Comparison"), that script _must_ be run on the system that has actually built the image you want to examine. It _cannot_ be a fresh YP install in which the image build has not yet been done and it is not possible to collect the data purely from a YP output image file that you happen to have - the information we need isn't available without context from the build system after having built the image. (If the original build for an image is unavailable, you can of course set up a new YP instance, configure it in exactly the same manner and build the same image and then you can run the script to gather the info; whether the info matches the original image built earlier depends on how closely you are able to match the configuration of course).